nemo_skills/pipeline/check_contamination.py

```diff
@@ -121,10 +130,28 @@ def check_contamination(
     get_random_port = server_gpus != 8 and not exclusive
 
     cluster_config = get_cluster_config(cluster, config_dir)
-    check_if_mounted(cluster_config, input_file)
```

Comment:
Can we keep this logic in the "else" part of `if check_mounted_paths`?

Reply:
This applies to all pipeline scripts

Reply:
Just a note, in places where `get_mounted_path` or `get_unmounted_path` exists, it implicitly calls check_if_mounted() - or it will raise an error saying the path is not mounted. So i removed the check_if_mounted() cause it was redundant in those sections